CODEOWNERS

```diff
@@ -6,7 +6,11 @@
 
 # The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
 
-* @fmvilas @derberg @dalelane @smoya @char0n @asyncapi-bot-eve @GreenRover
+# Specification code owners
+* @fmvilas @derberg @dalelane @smoya @char0n @GreenRover 
+
+# Tooling helpers
```

Comment:
What's the meaning of "Tooling helpers"? And whats gonna be the difference between you and the maintainers in https://github.com/asyncapi/spec-json-schemas/pull/522/files#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bR10 ? 🤔

Reply:
Personally, I wouldn't make such a distinction. I like what @derberg mentioned in https://github.com/asyncapi/spec-json-schemas/issues/513#issuecomment-2034861927: 

> we should treat spec-json-schemas as a tool, and focus more to have JSON Schema experts here

I believe all maintainers (atm, those at line 9) should be aware and able to review any PR.

Reply:
Alright 👍